### ISSUE TITLE
Financial: InsuranceProgram.process_claim adds uncovered loss to deductible_paid, conflating retained loss and SIR

### ISSUE THREAD

**Issue by AlexFiliakov** [AlexFiliakov/Ergodic-Insurance-Limits#934]
## Problem Statement
In \`insurance_program.py\`, the \`process_claim()\` method (lines 678-684) adds uncovered losses (amounts above the insurance program's coverage) to \`deductible_paid\`:

\`\`\`python
uncovered_loss = 0.0
total_covered = deductible_paid + insurance_recovery
if total_covered < claim_amount:
    uncovered_loss = claim_amount - total_covered
    # Company pays uncovered portion
    deductible_paid += uncovered_loss
\`\`\`

This conflates two distinct financial concepts:
1. **Self-Insured Retention (SIR) / Deductible**: The intentional risk retention below the attachment point.
2. **Uncovered Loss**: Losses above the program's maximum coverage (exhaustion point), due to gaps between layers, exhausted aggregate limits, or losses exceeding the top layer.

The \`ClaimResult\` reports both values but the final \`deductible_paid\` includes the uncovered portion. Downstream code in \`manufacturer_claims.py\` uses \`deductible_paid\` to determine the company's retained loss. Mixing these concepts makes it impossible to distinguish planned risk retention from uninsured catastrophic exposure.

Note: Issue #840 addresses this from the actuarial methodology perspective. This issue addresses the **financial accounting** impact — the ledger entries and income statement should distinguish between deductible expense (operating) and uninsured catastrophic loss (extraordinary/unusual item).

## v1.0 Impact
**Medium** — Financial analysis cannot distinguish planned risk retention from catastrophic uninsured exposure. Both are recorded as INSURANCE_LOSS in the ledger, preventing proper risk assessment and P&L categorization.

## Affected Code
- \`ergodic_insurance/insurance_program.py\`, lines 678-684 (\`process_claim\`)
- \`ergodic_insurance/manufacturer_claims.py\` — \`process_insurance_claim()\` uses deductible_paid for ledger entries

## Current Behavior
\`ClaimResult.deductible_paid\` = SIR + uncovered loss. Both are recorded as INSURANCE_LOSS.

## Expected Behavior (GAAP Reference)
Per **ASC 944-20-15-50**, the self-insured retention is a normal operating expense. Losses above program limits are different — per **ASC 450-20-50-1**, significant uninsured losses should be disclosed separately. The financial statements should distinguish:
- Deductible/SIR: Planned cost of risk, recorded as insurance operating expense
- Uncovered excess: Uninsured catastrophic loss, requiring separate disclosure

## Alternative Solutions
1. **Stop adding uncovered to deductible**: Remove lines 683-684. Report deductible_paid and uncovered_loss as separate, non-overlapping amounts. Update downstream code accordingly.
2. **Add separate ledger account**: Create an UNINSURED_LOSS account distinct from INSURANCE_LOSS.

## Recommended Approach
Option 1 — Remove the addition of uncovered_loss to deductible_paid in \`process_claim()\`. The ClaimResult already has a separate \`uncovered_loss\` field; let downstream code handle them independently. Update \`manufacturer_claims.py\` to record uncovered losses separately.

## Acceptance Criteria
- [ ] \`ClaimResult.deductible_paid\` equals only the SIR portion (not including uncovered loss)
- [ ] \`ClaimResult.uncovered_loss\` is handled separately in manufacturer_claims
- [ ] Total retained loss = deductible_paid + uncovered_loss (verified by assertion)
- [ ] Financial statements can distinguish deductible expense from uninsured loss

**Comment by AlexFiliakov** [comment]
## Issue Validation Review

**Status: CONFIRMED - Issue still exists on develop branch**

**Code Verification:**
- insurance_program.py lines 678-684: uncovered_loss is calculated as claim_amount - total_covered, then added to deductible_paid (line 684)
- deductible_paid originally represents the SIR/deductible amount, but after this addition it includes both the deductible AND the uncovered excess loss
- The ClaimResult.deductible_paid field then conflates two different concepts: retained loss within the SIR and uninsured loss above the program limits

**Priority Assessment:** Priority-medium is appropriate. The conflation makes it impossible to distinguish between planned retention (SIR) and unplanned excess loss in downstream analysis.

**Actionability:** Well-documented with clear code reference. The fix to separate deductible_paid from uncovered_loss in the return value is clean. Note that ClaimResult already has a separate uncovered_loss field, so the fix is to stop adding uncovered_loss to deductible_paid.

**Reviewer:** issue-validator | Batch: financial-accuracy

**Comment by AlexFiliakov** [comment]
**Issue Validation Review (2026-02-12)**

**Status**: Validated
**Priority Assessment**: Appropriate
**Completeness**: Actionable as-is
**Cross-references**: Part of financial-accuracy batch validation

Verified the referenced code. The described financial accuracy concern exists in the current codebase.
